components/MyTeamMembers.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="my-team__members">
+    <article v-for="member of members" :key="member" class="member-card">
+      <img class="member-card__photo" :src="member.photoUrl" alt="" />
+      <div class="member-card__job-title">
+        {{ member.jobTitle }}
+      </div>
+      <h2 class="member-card__name">
+        {{ member.name }}
+      </h2>
+    </article>
+  </div>
+</template>
+<script>
+export default {
+  name: 'MyTeamMembers',
```

Comment:
very very minor but this should be `MyTeamMember` without the s. 😊 A child component returns are based on the data available so this can return just one depending on the data and should be treated as a single child component that can be reused elsewhere. Ex: just a single card member component. 👍